mock/datastore.go

```diff
@@ -168,7 +169,7 @@ func (ds *DataStore) AddFilter(ctx context.Context, filterJob *models.Filter) (*
 }
 
 // AddFilterDimension represents the mocked version of creating a filter dimension to the datastore
-func (ds *DataStore) AddFilterDimension(ctx context.Context, filterID, name string, options []string, dimensions []models.Dimension, timestamp int64, eTagSelector string, currentFilter *models.Filter) (newETag string, err error) {
+func (ds *DataStore) AddFilterDimension(ctx context.Context, filterID, name string, options []string, dimensions []models.Dimension,timestamp primitive.Timestamp, eTagSelector string, currentFilter *models.Filter) (newETag string, err error) {
```

Comment:
Needs `go fmt` where the timestamp was replaced by the looks.

Reply:
resolved